bigquery_etl/dryrun.py

```diff
@@ -68,6 +68,9 @@
     "sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_amplitude_user_ids_v1/init.sql",  # noqa E501
     "sql/moz-fx-data-shared-prod/regrets_reporter/regrets_reporter_update/view.sql",
     "sql/moz-fx-data-shared-prod/revenue_derived/client_ltv_v1/query.sql",
+    "sql/moz-fx-data-shared-prod/monitoring/payload_bytes_decoded_structured/view.sql",
```

Comment:
does `all` need to be here?

Reply:
No, it should be dry-runnable once the authorized views it references are fully published.